### PR TITLE
Release php-ast 1.0.4

### PR DESCRIPTION
**Pull request by TysonAndre**

### CODE REVIEW

**Comment by TysonAndre** [comment]
Also, I'm a collaborator of the github repo, but not a maintainer of https://pecl.php.net/package/ast - When you have free time, could you update the release date, validate the release and publish it to PECL?

**Comment by nikic** [comment]
@TysonAndre I've added you as a maintainer on PECL.

**Comment by TysonAndre** [comment]
Sorry, it's been a while since I looked at the pecl website documentation. Regular maintainers of packages can't upload releases, just leads. When I try, I get the error message:

> **ERROR:**
> - You don't have permissions to upload this release.

https://pecl.php.net/release-upload.php

> Uploading new releases is restricted to each package's **lead** developer(s).

File: package.xml [hunk 1]
```diff
@@ -12,7 +12,13 @@
   <email>nikic@php.net</email>
   <active>yes</active>
  </lead>
- <date>2019-11-09</date>
+ <developer>
+  <name>Tyson Andre</name>
+  <user>tandre</user>
+  <email>tandre@php.net</email>
+  <active>yes</active>
+ </developer>
```
Comment (remicollet) [inline package.xml]: Developer is not enough to upload release on PECL.

**Comment by TysonAndre** [comment]
Okay, it's added to PECL. I'll update the details in package.xml in subsequent commits

**Pull request merged**